README.md

```diff
@@ -71,4 +71,29 @@ If you don't want to map anything, please leave the fields empty or with default
 
 7) Check the converted files on folder ``output/``.
 
-8) Import each converted file in Zephyr Scale using the default import option "Test Management for Jira".
\ No newline at end of file
+8) Import each converted file in Zephyr Scale using the default import option "Test Management for Jira".
+
+## Troubleshooting ##
+
+### Unexpected close tag ###
+Supposing that you have a file named myInput.xml on /input folder.
+During npm start you get the following error (or similar):
+```
+It was not possible to convert the content printed above
+look at the line specified by the error message:Unexpected close tag
```

Comment:
This look at the line sentence shouldn't be part of the ``` rich text block, or should it?

Reply:
ah sorry this is part of the log message but in the console they will see without it

Reply:
aah ok. Neverming. I see that you've added that log in the `app/testCaseXMLParser.js`. So ignore my previous comment :D